docs/sidebars.js

```diff
@@ -243,9 +243,14 @@ const sidebars = {
         },
         {
           type: 'doc',
-          label: 'Configuration Migrations',
+          label: 'Configuration migrations',
           id: 'reference/config-migration',
         },
+        {
+          type: 'doc',
+          label: 'SLSA adoption',
+          id: 'reference/slsa',
```

Comment:
> Which docs section works best? Is the reference section too hidden?

Seems fine to me, but I'd like to have links to it from the "Verify the CLI" page.